multiqc/modules/slamdunk/slamdunk.py

```diff
@@ -0,0 +1,832 @@
+        basic stats table at the top of the report """
+
+
+        headers = OrderedDict()
+        headers['sequenced'] = {
+            'title': 'Sequenced',
+            'description': '# sequenced reads',
+            'shared_key': 'slamdunk_reads',
+            'min': 0,
+            'format': '{:.f}'
+        }
+        headers['mapped'] = {
+            'title': 'Mapped',
+            'description': '# mapped reads',
+            'shared_key': 'slamdunk_reads',
+            'min': 0,
+            'format': '{:.f}'
+        }
+#         headers['deduplicated'] = {
+#             'title': 'Deduplicated',
+#             'description': '# deduplicated reads',
+#             'shared_key': 'slamdunk_reads',
+#             'min': 0,
+#             'format': '{:.f}'
+#         }
+        headers['mqfiltered'] = {
+            'title': 'MQ-Filtered',
+            'description': '# MQ-filtered reads',
+            'shared_key': 'slamdunk_reads',
+            'min': 0,
+            'format': '{:.f}'
+        }
+        headers['idfiltered'] = {
+            'title': 'Identity-Filtered',
+            'description': '# identity-filtered reads',
+            'shared_key': 'slamdunk_reads',
+            'min': 0,
+            'format': '{:.f}'
+        }
+        headers['nmfiltered'] = {
+            'title': 'NM-Filtered',
+            'description': '# NM-filtered reads',
+            'shared_key': 'slamdunk_reads',
+            'min': 0,
+            'format': '{:.f}'
+        }
+        headers['multimapper'] = {
+            'title': 'Multimap-Filtered',
+            'description': '# multimap-filtered reads',
+            'shared_key': 'slamdunk_reads',
+            'min': 0,
+            'format': '{:.f}'
+        }
+        headers['retained'] = {
+            'title': 'Retained',
+            'description': '# retained reads after filtering',
+            'shared_key': 'slamdunk_reads',
+            'min': 0,
+            'format': '{:.f}'
+        }
+        
+        if (extendedSummary) :
+            headers['counted'] = {
+                'title': 'Counted',
+                'description': '# reads counted within 3\'UTRs',
+                'shared_key': 'slamdunk_reads',
+                'min': 0,
+                'format': '{:.f}'
+            }
+        self.general_stats_addcols(self.slamdunk_data, headers)
+        
+    def slamdunkOverallRatesPlot (self):
+        """ Generate the overall rates plot """
+
+        pconfig = {
+            'id': 'overallratesplot',
+            'title': 'Overall conversion rates in reads',
+            'cpswitch' : False,
+            'cpswitch_c_active': False,
+            'stacking' : 'normal',
+            'data_labels': [
+                "Plus Strand +",
+                "Minus Strand -", 
+            ]
+        }
+        
+        cats = [OrderedDict(), OrderedDict()]
+        
+        cats[0]['T>C'] = {
+            'name': 'T>C',
```

Comment:
If the `name` is the same as the dictionary key, you don't need to specify it as this will be the default anyway.